package/testing/Scripts/get-zeek-env

```diff
@@ -30,7 +30,7 @@ else
         # Combine the local tree and the system-wide path. This allows
         # us to test on a local build or an installation made via zkg,
         # which squirrels away the build. --cpk
```

Comment:
Since you are touching this already, could you remove the `--cpk` on the comment?